test/unit/math/mix/prob/normal_cdf_test.cpp

```diff
@@ -0,0 +1,28 @@
+#include <test/unit/math/test_ad.hpp>
+#include <limits>
+
+TEST(mathMixScalFun, normal_cdf_derivatives) {
```

Comment:
Are there mix tests for normal_cdf_log, normal_ccdf_log too?

Reply:
`normal_cdf_log_test` was already there, I'm going to add `normal_ccdf_log_test` now.